### PR TITLE
Update package name, application icon for Castanets.

### PR DESCRIPTION
**Pull request by suyambulingamrm**
This change updates the package name, application name and uses
Castanets logo as application icon.

Signed-off-by: suyambu.rm <suyambu.rm@samsung.com>

### CODE REVIEW

File: chrome/android/BUILD.gn [hunk 1]
```diff
@@ -22,7 +22,7 @@ import("channel.gni")
 import("java_sources.gni")
 import("static_initializers.gni")
 
-manifest_package = "org.chromium.chrome"
+manifest_package = "com.samsung.castanets"
```
Comment (sm-venugopal) [inline chrome/android/BUILD.gn]: Can use "enable_castanets" and retain the original code.

**Review by Chikdolman — APPROVED** [review]
LGTM

File: chrome/android/BUILD.gn [hunk 2]
```diff
@@ -22,7 +22,11 @@ import("channel.gni")
 import("java_sources.gni")
 import("static_initializers.gni")
 
-manifest_package = "org.chromium.chrome"
+if (enable_castanets) {
+  manifest_package = "com.samsung.castanets"
```
Comment (younghajung) [inline chrome/android/BUILD.gn]: How about "com.samsung.android.castanets"?

Comment (suyambulingamrm) [reply chrome/android/BUILD.gn]: Updated on latest changeset.

**Review by younghajung — APPROVED** [review]
LGTM

**Pull request merged**